mm2src/mm2_main/src/lp_swap/maker_swap_v2.rs

```diff
@@ -822,8 +811,8 @@ impl<MakerCoin: MmCoin + MakerCoinSwapOpsV2, TakerCoin: MmCoin + TakerCoinSwapOp
             },
         };
 
-        let taker_payment_spend_trade_fee = match state_machine.taker_coin.get_receiver_trade_fee(stage).compat().await
-        {
+        // FIXME: Fix the naming of this variable, and the state machine variables.
```

Comment:
I suppose naming was fixed in this https://github.com/KomodoPlatform/komodo-defi-framework/pull/2109/commits/0a4248cc67e6e2d7874b87e7ea0f5fd750324b76 ?

Reply:
Indeed. Thanks!